test/unit/app/models/attachment_data_test.rb

```diff
@@ -444,11 +437,4 @@ class AttachmentDataTest < ActiveSupport::TestCase
 
     assert_not attachment_data.all_asset_variants_uploaded?
   end
-
-  test "#all_asset_variants_uploaded? returns false if some asset variants are missing" do
```

Comment:
Covered in the test above (about the original, which is technically a variant). Happy to delete ✅